frontend/src/pages/modelRegistry/screens/RegisterModel/RegisterModelErrors.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { Alert, AlertActionCloseButton, StackItem } from '@patternfly/react-core';
+
+type RegisterModelErrorProp = {
+  submitLabel: string;
+  submitError: Error;
+  errorName?: string;
+  versionName?: string;
+  modelName?: string;
+};
+
+const RegisterModelErrors: React.FC<RegisterModelErrorProp> = ({
+  submitLabel,
+  submitError,
+  errorName,
+  versionName,
+  modelName,
+}) => {
+  const [showAlert, setShowAlert] = React.useState<boolean>(true);
+
+  if (submitLabel === 'Register model' && errorName === 'modelVersion') {
+    return (
+      <>
+        {showAlert && (
+          <StackItem>
+            <Alert
+              isInline
+              variant="success"
+              title={`${modelName} model registered`}
```

Comment:
you'll need to make sure that modelName and versionName are always defined (or some other default value is given) for these uses in the template